pytorch_lightning/callbacks/progress/progress.py

```diff
@@ -206,6 +209,54 @@ def init_test_tqdm(self) -> tqdm:
         )
         return bar
 
+    def get_progress_bar_dict(
+        self, trainer: "pl.Trainer", pl_module: "pl.LightningModule"
+    ) -> Dict[str, Union[int, str]]:
```

Comment:
@daniellepintz @kaushikb11 - should this go in the base progress bar? Would this be useful for the RichProgressBar too? I don't think this is tqdm specific